docs/gettingStarted/Arena-Environment-Guide.md

```diff
@@ -0,0 +1,310 @@
+- `randomizeArenas` a `bool`, defines whether the arena will be randomized between episodes. If set to `true`, the arena will be randomized between the defined Arenas in the configuration file. If set to `false`, the order to which the arenas are spawned are sequential and top-to-bottom as specified in the configuration file. This parameter is set to `false` by default.
+- `showNotification` a `bool`, defines whether the player will receive a notification at the end of an episode. If set to `true`, the player will be shown a notification at the end of an episode for approximately 2.5 seconds, then move on to the next episode (arena). If set to `false`, the agent will not receive a notification at the end of an episode and episode-to-episode termination is back-to-back. This parameter is set to `false` by default.
+- `blackouts` a `list`, defines the frames at which the lights are on or off during an episode. If omitted, the lights will be on for the entire episode. For more information on blackouts, [see here](#blackouts)
+
+**N.B:** These parameters are optional (except `t` and `pass_mark`) and can be omitted from the configuration file. If omitted, the default values will be used, which are explained in detail in our [YAML Config Syntax](/docs/configGuide/YAML-Config-Syntax.md) guide.
+
+## The Agent
+
+The agent is the main character in the arena, for playing and training. It is a spherical animal with a set of controls that can be used to move it around the arena. The agent can be configured to have a set of different skins, which can be specified in the configuration file, under it's parameters. The agent has a set of controls that can be used to move it around the arena. 
+
+The controls are as follows:
+
+- `W` - move forward
+- `A` - move left
+- `S` - move backward
+- `D` - move right
+- `C` - change camera perspective (first-person, third-person, eagle-view, only if `canChangePerspective` is `true`)
+- `R` - reset the arena (cycles to the next episode if `canResetEpisode` is `true`)
+- `Q` - quit (exits the application upon press)
+
+<table>
+  <tr>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-hedgehog.png" width="500"/>
+    <p>Hedgehog</p></td>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-panda.png" width="500"/><p>Panda</p></td>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-pig.png" width="500"/><p>Pig</p></td>
+</table>
+
+### Agent HUD (Heads-Up Display)
+
+The agent has a HUD (_Heads-Up Display_) that displays the following information per episode:
```

Comment:
nit: I would mention if this is / isn't included when training

Reply:
done :)